### PR TITLE
feat: Add alarm on high disk usage

### LINKED ISSUE #172: feat: Terminate runner instance when disk usage is high
Terminates runner instance with low disk space in order to avoid run-time error.

### Use Case

As time goes by, free space of EBS volume is getting smaller. It causes "no space left on device" error at run-time. Gitlab runners have low free EBS space should be terminated before that.

### Proposed Solution

Uses SNS topic and Cloudwatch alarm to call a lambda function, that terminates runner instance on low disk space using boto3.

### Other

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by gotodeploy**
Fixes #172

### CODE REVIEW

File: src/integ.gitlab-runner-autoscaling.ts [hunk 1]
```diff
@@ -21,8 +23,33 @@ const runner = new GitlabRunnerAutoscaling(stack, 'TestRunnerAutoscaling', {
...
+      Threshold: 50,
+    },
+  ],
 });
 
+/**
+ * Scheduled scaling
+ * https://docs.aws.amazon.com/cdk/api/latest/docs/aws-autoscaling-readme.html#scheduled-scaling
+ */
+runner.autoscalingGroup.scaleOnSchedule('StopOnWeekends', {
```
Comment (guan840912) [inline src/integ.gitlab-runner-autoscaling.ts:42]: Let's make it become to runner's function ?! like this:
```ts
runner.stopOnWeekends({
  schedule: { weekDay: 'fri', hour: '18', minute: '0' },
  minCapacity: 0,
  maxCapacity: 0,
}
```

File: src/integ.gitlab-runner-autoscaling.ts [hunk 2]
```diff
@@ -21,8 +23,33 @@ const runner = new GitlabRunnerAutoscaling(stack, 'TestRunnerAutoscaling', {
...
+  schedule: asg.Schedule.cron({ weekDay: 'fri', hour: '18', minute: '0' }),
+  minCapacity: 0,
+  maxCapacity: 0,
+});
+
+runner.autoscalingGroup.scaleOnSchedule('WorkOnWeekdays', {
```
Comment (guan840912) [inline src/integ.gitlab-runner-autoscaling.ts:48]: Let's make it become to  runner's function ?!  like this:
```ts
runner.workOnWeekdays({
  schedule: { weekDay: 'fri', hour: '18', minute: '0' },
  minCapacity: defaultCapacity,
  maxCapacity: defaultCapacity,
}
```

**Review by guan840912 — CHANGES_REQUESTED** [review]

File: src/integ.gitlab-runner-autoscaling.ts (hunk 2 above)
Comment (gotodeploy) [reply src/integ.gitlab-runner-autoscaling.ts:48]: @guan840912 I think this should be imprelemented in another issue.

Comment (guan840912) [reply src/integ.gitlab-runner-autoscaling.ts:48]: ok LGTM

File: src/integ.gitlab-runner-autoscaling.ts (hunk 1 above)
Comment (guan840912) [reply src/integ.gitlab-runner-autoscaling.ts:42]: ok LGTM

**Pull request merged**